sematic/config.py

```diff
@@ -108,6 +110,13 @@ def server_pid_file_path(self):
 class UserOverrideConfig(Config):
     @property
     def server_url(self) -> str:
+        # environment vars should take precedence over whatever is in the
+        # users settings file.
+        server_address = os.environ.get("SEMATIC_SERVER_ADDRESS", None)
```

Comment:
Why not use `get_user_settings` since it already includes an env var override mechanism?

This way users can either do `$ SEMATIC_SERVER_ADDRESS=0.0.0.0 sematic start` or set it permanently with `$ sematic settings set SEMATIC_SERVER_ADDRESS 0.0.0.0`.

Reply:
You'll note below that `UserOverrideConfig` falls back to `get_user_settings(SettingsVar.SEMATIC_API_ADDRESS)` if the env vars are not set. Basically this is just here so that if you use the env vars they take precedence over whatever is in your settings file.

Reply:
> the env vars they take precedence over whatever is in your settings file

That is my point, this mechanism already exists in `get_user_settings`. All named settings can be overridden from env vars.